### PR TITLE
Project/entry review

### PR DESCRIPTION
**Pull request by AdityaKhatri**
Depends on https://github.com/the-deep/server/pull/703

### CODE REVIEW

File: src/components/general/EntryCommentFormForModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        setFaramValues(newFaramValues);
+        setFaramErrors(newFaramErrors);
+    }, []);
+
+    const handleFaramValidationSuccess = React.useCallback((_, newFaramValues: T) => {
+        onValidationSuccess(newFaramValues);
+    }, [onValidationSuccess]);
+
+    return (
+        <Faram
```
Comment (subinasr) [inline src/components/general/EntryCommentFormForModal/index.tsx:72]: Why aren't we using the new form here?

File: src/components/general/EntryCommentModal/CommentList/EditCommentForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        onEditCancel,
+    } = props;
+
+    const [pristine, setPristine] = useState<boolean>(true);
+    const [faramErrors, setFaramErrors] = useState<FaramErrors>();
+    const [faramValues, setFaramValues] = useState<Comment | undefined>({
+        text: comment.textHistory[0]?.text,
+        commentType: comment.commentType,
+        mentionedUsers: comment.mentionedUsers,
+    });
```
Comment (subinasr) [inline src/components/general/EntryCommentModal/CommentList/EditCommentForm/index.tsx:45]: Why are we defining type like this and not outside the function (using interface) like every other time? Any specific reasons

File: src/components/general/EntryCommentModal/CommentList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            />
+            <div
+                className={styles.content}
+            >
+                <div className={styles.heading}>
+                    <span className={styles.detail}>
+                        <span className={styles.name}>{createdByDetails?.name}</span>
+                        &nbsp;
+                        {_ts('entryReview', 'commentType', { commentType: commentTypeToTextMap[commentType] })}
+                        &nbsp;
```
Comment (subinasr) [inline src/components/general/EntryCommentModal/CommentList/index.tsx:85]: Why not use another span and add padding instead of &nbsp?

**Comment by AdityaKhatri** [comment]
We implemented this in new design

**Pull request closed without merging**